bokeh/models/glyphs.py

```diff
@@ -737,580 +1027,835 @@ class MultiPolygons(Glyph):
-    """)
+    """
+    )
 
-    bottom = NumberSpec(default=0, help="""
+    bottom = NumberSpec(
+        default=0,
+        help="""
     The y-coordinates of the bottom edges.
-    """)
+    """,
+    )
 
-    top = NumberSpec(help="""
+    top = NumberSpec(
+        help="""
     The y-coordinates of the top edges.
-    """)
+    """
+    )
 
-    line_props = Include(LineProps, use_prefix=False, help="""
+    line_props = Include(
+        LineProps,
+        use_prefix=False,
+        help="""
     The %s values for the vertical bars.
-    """)
+    """,
+    )
 
-    fill_props = Include(FillProps, use_prefix=False, help="""
+    fill_props = Include(
+        FillProps,
+        use_prefix=False,
+        help="""
     The %s values for the vertical bars.
-    """)
+    """,
+    )
 
-    hatch_props = Include(HatchProps, use_prefix=False, help="""
+    hatch_props = Include(
+        HatchProps,
+        use_prefix=False,
+        help="""
     The %s values for the vertical bars.
-    """)
+    """,
+    )
+
 
-class Wedge(XYGlyph):
-    ''' Render wedges.
+class Wedge(LineGlyph, FillGlyph):
+    """ Render wedges.
 
-    '''
+
+    """
 
     __example__ = "examples/reference/models/Wedge.py"
 
     # a canonical order for positional args that can be used for any
     # functions derived from this class
-    _args = ('x', 'y', 'radius', 'start_angle', 'end_angle', 'direction')
+    _args = ("x", "y", "radius", "start_angle", "end_angle", "direction")
 
-    x = NumberSpec(help="""
+    x = NumberSpec(
+        help="""
     The x-coordinates of the points of the wedges.
-    """)
+    """
+    )
 
-    y = NumberSpec(help="""
+    y = NumberSpec(
+        help="""
     The y-coordinates of the points of the wedges.
-    """)
+    """
+    )
 
-    radius = DistanceSpec(help="""
+    radius = DistanceSpec(
+        help="""
     Radii of the wedges.
-    """)
+    """
+    )
 
-    start_angle = AngleSpec(help="""
+    start_angle = AngleSpec(
+        help="""
     The angles to start the wedges, as measured from the horizontal.
-    """)
+    """
+    )
 
-    end_angle = AngleSpec(help="""
+    end_angle = AngleSpec(
+        help="""
     The angles to end the wedges, as measured from the horizontal.
-    """)
+    """
+    )
 
-    direction = Enum(Direction, default='anticlock', help="""
+    direction = Enum(
+        Direction,
+        default="anticlock",
+        help="""
     Which direction to stroke between the start and end angles.
-    """)
+    """,
+    )
 
-    line_props = Include(LineProps, use_prefix=False, help="""
+    line_props = Include(
+        LineProps,
+        use_prefix=False,
+        help="""
     The %s values for the wedges.
-    """)
+    """,
+    )
 
-    fill_props = Include(FillProps, use_prefix=False, help="""
+    fill_props = Include(
+        FillProps,
+        use_prefix=False,
+        help="""
     The %s values for the wedges.
-    """)
+    """,
+    )
```

Comment:
Why all those unrelated changes? How we are supposed to know what you actually changed in this file? You will have to revert unrelated changes and leave the minimum number of changes to resolve the issues in question.

Reply:
I applied black formatting unconsciously, I hopefully cleaned it up and it is clearer now.

Reply:
Thanks. It's much clearer now to see what's the extent of your changes.